public/app/features/integrated-alerting/components/AlertRules/AlertRulesActions/AlertRulesActions.test.tsx

```diff
@@ -4,16 +4,62 @@ import { act } from 'react-dom/test-utils';
 import { dataQa } from '@percona/platform-core';
 import { AlertRulesActions } from './AlertRulesActions';
 import { AlertRulesProvider } from '../AlertRules.provider';
-import { formattedRulesStubs, alertRulesContextStub } from '../__mocks__/alertRulesStubs';
+import { rulesStubs, formattedRulesStubs, alertRulesContextStub } from '../__mocks__/alertRulesStubs';
+import { AlertRulesService } from '../AlertRules.service';
+import { Messages } from './AlertRulesActions.messages';
 
 jest.mock('../AlertRules.service');
 jest.mock('app/core/app_events');
 
+const alertRulesServiceCreate = jest.spyOn(AlertRulesService, 'create');
+
 describe('AlertRulesActions', () => {
   afterEach(() => {
     jest.clearAllMocks();
   });
 
+  it('calls the API to crate an alert rule on copy', async () => {
+    const testRule = rulesStubs[1];
+
+    const wrapper = mount(<AlertRulesActions alertRule={formattedRulesStubs[1]} />);
+
+    const expectedResult = {
+      ...testRule,
+      disabled: true,
+      summary: `${Messages.copyOf} ${testRule.summary}`,
+      custom_labels: undefined as any,
```

Comment:
Why not to remove `custom_labels` since is nor required?

Reply:
I'd have to implement the logic to remove it if it's undefined and I don't think it makes sense :) If you're talking just about the test, then it has to be explicit

Reply:
Ok got it!